doc/book/examples/sources/basics/publisher.move

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// A simple package that defines an OTW and claims a `Publisher`
+/// object for the sender.
+module examples::owner {
+    use sui::tx_context::TxContext;
+    use sui::package;
+
+    /// OTW is a struct with only `drop` and is named
+    /// after the module - but uppercased. See "One Time
+    /// Witness" page for more details.
+    struct OWNER has drop {}
+
+    /// Some other type to use in a dummy check
+    struct ThisType {}
+
+    /// After the module is published, the sender will receive
+    /// a `Publisher` object. Which can be used to set Display
+    /// or manage the transfer policies in the `Kiosk` system.
+    fun init(otw: OWNER, ctx: &mut TxContext) {
+        package::claim_and_keep(otw, ctx)
+    }
+}
+
+/// A module that utilizes the `Publisher` object to give a token
+/// of appreciation and a `TypeOwnerCap` for the owned type.
+module examples::type_owner {
+    use sui::object::{Self, UID};
+    use sui::tx_context::TxContext;
+    use sui::package::{Self, Publisher};
+
+    /// Trying to claim ownership of a type with a wrong `Publisher`.
+    const ENotOwner: u64 = 0;
+
+    /// A capability granted to those who want an "objective"
+    /// confirmation of their ownership :)
+    struct TypeOwnerCap<phantom T> has key, store {
```

Comment:
Would this be a utility type that we would provide, or do we want folks to roll their own?

Reply:
It's really hard to come up with a real production-ready case for the Publisher. It unblocks generic modules (like Collectible, Royalty, Kiosk, Display etc) but I don't know a single simple app that would require proof of type ownership.